lang/java/reef-bridge-java/src/main/java/org/apache/reef/javabridge/AllocatedEvaluatorBridge.java

```diff
@@ -139,7 +139,18 @@ public void submitContextAndServiceAndTaskString(
    * Gets the serialized evaluator descriptor from the Java allocated evaluator.
    * @return the serialized evaluator descriptor.
    */
-  public String getEvaluatorDescriptorSring() {
+  public String getEvaluatorDescriptorString() {
+    final String descriptorString =
+            Utilities.getEvaluatorDescriptorString(jallocatedEvaluator.getEvaluatorDescriptor());
+    LOG.log(Level.INFO, "allocated evaluator - serialized evaluator descriptor: " + descriptorString);
+    return descriptorString;
+  }
+
+  /**
+   * @deprecated in 0.14. Use getEvaluatorDescriptorString instead.
+   */
+  @Deprecated
+  public String getEvaluatorDescriptorSting() {
```

Comment:
One of these methods is used in our .cpp bridge code. Please update the usage to the new spelling, so that it doesn't break when the deprecated method is deleted - it doesn't show as usage.